Update images.md (#23130)
kubernetes/website: content/en/docs/concepts/containers/images.md

This approach is suitable if you can control node configuration.

{{< note >}}

> Kubernetes as is only supports the `auths` and `HttpHeaders` section in Docker configuration.

Reviewer: I would rephrase this slightly:
```suggestion
Default Kubernetes only supports the `auths` and `HttpHeaders` section in Docker configuration.
```
